modules/installation-vsphere-machines.adoc

```diff
@@ -154,33 +144,59 @@ specify values on the *Customize template* tab.
 +
 [IMPORTANT]
 ====
-If you plan to add more compute machines to your cluster after you finish
+If you plan to add more worker machines to your cluster after you finish
 installation, do not delete this template.
 ====
 
 . After the template deploys, deploy a VM for a machine in the cluster.
-.. Right-click the template's name and click *Clone* -> *Clone to Virtual Machine*.
+.. Right-click the template name and click *Clone* -> *Clone to Virtual Machine*.
 .. On the *Select a name and folder* tab, specify a name for the VM. You might
-include the machine type in the name, such as `control-plane-0` or `compute-1`.
+include the machine type in the name, such as `control-plane-0` or `worker-1`.
 .. On the *Select a name and folder* tab, select the name of the folder that
 you created for the cluster.
-.. On the *Select a compute resource* tab, select the name of a host in your
-datacenter.
+.. On the *Select a compute resource* tab, select the name of a host in your datacenter.
++
+For a bootstrap machine, specify the URL of the bootstrap Ignition config file that you hosted.
++
 .. Optional: On the *Select storage* tab, customize the storage options.
 .. On the *Select clone options*, select
 *Customize this virtual machine's hardware*.
 .. On the *Customize hardware* tab, click *VM Options* -> *Advanced*.
+*** Optional: Override default DHCP networking in vSphere. To enable static IP networking:
```

Comment:
We don't have this currently configured in CI and it probably should be.

Reply:
Is the wording okay here? It's in OCP 4.6 release notes already.